osgtest/tests/test_580_gfal2util.py

```diff
@@ -13,7 +13,8 @@ class TestGFAL2Util(osgunittest.OSGTestCase):
 
     def setUp(self):
         self.skip_ok_unless(core.state['proxy.valid'] or core.state['voms.got-proxy'], "No proxy")
-        core.skip_ok_unless_installed('gfal2-util', 'gfal2-plugin-file')
+        core.skip_ok_unless_installed('gfal2-util-scripts', 'gfal2-plugin-file')
+        core.skip_ok_unless_one_installed('python2-gfal2-util', 'python3-gfal2-util')
```

Comment:
So what you apparently figured out, but was not obvious to me from the commit message or PR body, is that the new `gfal2-util-scripts` package contains the (non-python) shell scripts from `gfal2-util`, and that the new `python[23]-gfal2-util` packages now pull in `gfal2-util-scripts`.

But yeah with that in mind the updated logic looks good!

Reply:
I'll update the commit message to clarify!